lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -325,6 +339,13 @@ def get_unread_comment_count(self, obj):
             return obj["unread_comments_count"] + 1
         return obj["unread_comments_count"]
 
+    def get_preview_body(self, obj):
+        """
+        Returns a cleaned and truncated version of the thread's body to display in a
```

Comment:
can you explain where this text will be used in the UI? what is the benefit of manipulating the rendered body from the backend?

Reply:
On the list of posts on the left side, there is a single sentence preview of each post. However, since posts can be formatted in all manner of ways, this means that the preview text on the left can look inconsisted across posts, and is hard to truncate. 

We can either clean the HTML on the client side by using a library for it, (not sure how much this will add to the app size) or we can do it on the server-side where we already have the functionality ready.

Reply:
got it. I agree to truncate it from the backend as it's just _10 words_ and won't increase the size of the API request.